module/tasks/openchain.tasks.ps1

```diff
@@ -0,0 +1,156 @@
+# <copyright file="openchain.tasks.ps1" company="Endjin Limited">
+# Copyright (c) Endjin Limited. All rights reserved.
+# </copyright>
+

```

Comment:
I'm not sure if this is the convention with invoke-build, but I'm wondering if you can use `param` blocks or similar to better define what the actual parameters are?

Reply:
We can't use `param` blocks in these files, just in the entrypoint script (e.g. `build.ps1`).  We have the strongly-typed configuration model that is proposed as a solution [this issue](https://github.com/zerofailed/ZeroFailed/issues/7).